### DISCUSSION TITLE
How to catch-all requests in for specific path?

### DISCUSSION THREAD

**Discussion by devhus** [nuxt/nuxt#16785] [Questions]
want to know how to catch all requests coming within a specific route using one file, for example 
```
- server
-- routes
--- isb.ts
```
How to make all the coming requests to ``nuxt-ssr-example.com/isb``, ``nuxt-ssr-example.com/isb/*.*`` run the ``isb.ts`` route file?

**Comment by sevillaarvin** [comment]
How about the catch-all route? https://v3.nuxtjs.org/guide/directory-structure/server#catch-all-route

**Reply by devhus** [reply]
> How about the catch-all route? https://v3.nuxtjs.org/guide/directory-structure/server#catch-all-route

I have tried that but it wasn't working,  and it seems like ''[...].ts'' file does not have any effect in my project for some reason

**Reply by sevillaarvin** [reply]
How about something like this?

https://stackblitz.com/edit/github-g54qsw?file=app.vue,server%2Froutes%2Fisb.ts,server%2Froutes%2Fisb%2F[...].ts

**Reply by devhus** [reply]
> How about something like this?
> 
> [https://stackblitz.com/edit/github-g54qsw?file=app.vue,server%2Froutes%2Fisb.ts,server%2Froutes%2Fisb%2F[...].ts](https://stackblitz.com/edit/github-g54qsw?file=app.vue,server%2Froutes%2Fisb.ts,server%2Froutes%2Fisb%2F%5B...%5D.ts)

Thank you
